docs/extensions-plugins/obsidian.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: Pieces for Developers Obsidian Plugin
+description: Troubleshoot issues or reach out to Pieces Support for assistance with the Pieces for Developers Obsidian Plugin
+---
+
+<link
+  rel="canonical"
+  href="https://docs.pieces.app/extensions-plugins/obsidian"
+/>
+
+export const InstallButton = (props) => {
+  return (
+    <a
+      style={{ textDecoration: "none" }}
+      target="_blank"
+      href="{{ links.extensions.obsidian }}"
+    >
+      <div style={{ maxWidth: "fit-content", minWidth: "fit-content" }}>
+        <p
+          style={{
+            padding: "8px 20px",
+            borderRadius: "8px",
+            border: "1px solid #f3f5f6",
+            color: "white",
+            fontWeight: "500",
+            background: "black",
+            marginBottom: "8px",
+            marginTop: "3px",
+            marginRight: "16px",
+          }}
+        >
+          {props.children}
+        </p>
+      </div>
+    </a>
+  );
+};
+
+export const ReleasePill = (props) => {
+  return (
+    <a
+      style={{ textDecoration: "none" }}
+      target="_blank"
+      href={"https://code.pieces.app/updates"}
+    >
+      <div style={{ maxWidth: "fit-content", minWidth: "fit-content" }}>
+        <p
+          class="dark:text-white text-black"
+          style={{
+            padding: "8px 20px",
+            borderRadius: "8px",
+            border: "1px solid #f3f5f6",
+            backgroundColor: "transparent",
+            fontWeight: "500",
+            marginBottom: "8px",
+            marginTop: "3px",
+            marginRight: "16px",
+          }}
+        >
+          {props.children}
+        </p>
+      </div>
+    </a>
+  );
+};
+
+export const GridFlexRow = (props) => {
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "start",
+        flexWrap: "wrap",
+      }}
+    >
+      {props.children}
+    </div>
+  );
+};
+
+export const HeaderLogoImage = (props) => {
+  return (
+    <img
+      style={{ marginLeft: "0px", height: "30%", marginRight: "12px" }}
+      src={"/assets/obsidian.png"}
+    />
+  );
+};
+
+<GridFlexRow>
+  <h1>Pieces Obsidian Plugin</h1>
+</GridFlexRow>
+
+<br></br>
+
+<GridFlexRow>
+  <InstallButton>Get the Obsidian Plugin</InstallButton>
+  <ReleasePill children="Current Version: {{ versions.obsidian }}"></ReleasePill>
+</GridFlexRow>
+
+<br></br>
+
+# Your Guide to Getting Started with Pieces for Obsidian
+
+_Pieces for Obsidian Pieces captures **NO identifiable user data**. Our **Local only** architecture means your notes never have to leave your device!_
+
+_The <a target="_blank" href="{{ links.website.pfd_desktop_install }}">Pieces Suite</a> must be downloaded and installed separately from the Obsidian Plugin in order for the Plugin to work properly._
+
+## Save your Snippets
+
+- To save a snippet, highlight the text, right-click, and select "Save to Pieces."
+  **OR**
```

Comment:
<img width="550" alt="Screenshot 2023-06-05 at 3 39 34 PM" src="https://github.com/pieces-app/documentation/assets/55861512/a870380f-b1f0-45fb-9c29-272ad99011d0">

adding **OR** without the proper leading line and trailing line will make it wrap like the image above, try to keep individual lines seperate from other lines like this: 

```markdown
# First Line 

second line that is right underneath

### third line 

also this seems to work for me two as long as the types are different: 

# Heading text here
subheader text that will be on a new line